src/forms/CurrentBenefits.js

```diff
@@ -23,45 +25,48 @@ import { allBenefitOrders } from '../programs/allBenefitOrders';
  */
 class CurrentBenefitsContent extends React.Component {
   handleRadioChange = (event, inputProps) => {
-    const benefitName = inputProps.name;
-
-    const route = 'benefits';
-
-    const value = cloneDeep(this.props.current.benefits);
+    const benefitName = inputProps.name,
+          route       = `benefits`,
+          value       = cloneDeep(this.props.current.benefits);
 
     if (inputProps.value) {
       value.push(benefitName);
       value.sort(
         // Make sure benefits are in the correct order
         (a, b) => {
-          const aIndex = this.props.benefits.indexOf(a);
-          const bIndex = this.props.benefits.indexOf(b);
+          // @todo Is this code right? Don't we need `.current` in here?
```

Comment:
`props.benefits` here is a list of all available benefits, rather than the client's benefits, so no `current` needed.

Reply:
It might be good to remove this comment, just so we're not chasing geese when we pick this up later (assuming your concerns are assuaged).